ERROR - 2024-10-01 02:42:27 --> mysqli_sql_exception: Unknown column 'type' in 'settings' in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('ALTER TABLE `se...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('ALTER TABLE `se...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('ALTER TABLE `se...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(772): CodeIgniter\Database\BaseConnection->query('ALTER TABLE `se...')
#4 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\settings\src\Database\Migrations\2021-11-14-143905_AddContextColumn.php(23): CodeIgniter\Database\Forge->addColumn('settings', Array)
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): CodeIgniter\Settings\Database\Migrations\AddContextColumn->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-10-01 02:44:58 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-10-01 02:47:36 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
INFO - 2024-10-01 02:48:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 02:48:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-10-01 02:53:31 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-10-01 02:57:51 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-10-01 03:03:08 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2024-10-01 03:07:18 --> mysqli_sql_exception: Table 'settings' already exists in C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(312): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\web\Runov_Litbang\app\Database\Migrations\2024-09-30-081033_CreateSettings.php(22): CodeIgniter\Database\Forge->createTable('settings')
#5 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(867): App\Database\Migrations\CreateSettings->up()
#6 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\web\Runov_Litbang\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\web\Runov_Litbang\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-10-01 03:08:40 --> ErrorException: Cannot declare class CodeIgniter\Shield\Database\Migrations\CreateAuthTables, because the name is already in use
[Method: CLI, Route: migrate]
in APPPATH\Database\Migrations\2024-10-01-025602_CreateAuthTables.php on line 20.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-10-01 03:10:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:10:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:10:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:10:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:11:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:12:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:13:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-01 03:14:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
